test/chaat/app_test.clj

```diff
@@ -0,0 +1,12 @@
+(ns chaat.app-test
+  (:require [chaat.fixture :as fixture]
+            [clojure.test :refer :all]
+            [clj-http.client :as client]))
+
+(use-fixtures :each fixture/test-fixture)
+
+(deftest end-end-test
```

Comment:
question: name?

Reply:
oops I was going for something that indicated end to end test. Updated to `health-check-test`.